test/components/tree-view.spec.tsx

```diff
@@ -128,314 +116,376 @@ describe('<TreeView />', () => {
-                const {select, waitForDom} = clientRenderer.render(<TreeViewDemo />);
+                const {driver, waitForDom} = clientRenderer.render(
+                    <TreeViewDemo />
+                ).withDriver(TreeViewDemoDriver);
+
+                const {treeView} = driver;
 
                 const nodeChildren = treeData[0].children;
 
-                selectItemWithLabel(select, treeData[0].label);
-                expandItemWithLabel(select, treeData[0].label);
+                treeView.selectItem(treeData[0].label);
+                treeView.toggleItem(treeData[0].label);
 
-                await waitForDom(() => expect(select(getTreeItem(nodeChildren![1].label))).to.be.present());
+                await waitForDom(() => expect(treeView.getItem(nodeChildren![1].label)).to.be.present());
 
-                simulate.keyDown(select('TREE_VIEW_DEMO', 'TREE_VIEW'), {keyCode: KeyCodes.LEFT});
-                await waitForDom(() => expect(select(getTreeItem(nodeChildren![1].label))).to.be.absent());
+                treeView.pressKey(keycode('left'));
 
-                simulate.keyDown(select('TREE_VIEW_DEMO', 'TREE_VIEW'), {keyCode: KeyCodes.RIGHT});
+                await waitForDom(() => expect(treeView.getItem(nodeChildren![1].label)).to.be.absent());
 
-                return waitForDom(() => expect(select(getTreeItem(nodeChildren![1].label))).to.be.present());
+                treeView.pressKey(keycode('right'));
+
+                await waitForDom(() => expect(treeView.getItem(nodeChildren![1].label)).to.be.present());
             });
 
             it('returns to parent if there is after collapsing the element if possible when left is clicked',
                 async () => {
-                    const {select, waitForDom} = clientRenderer.render(<TreeViewDemo />);
+                    const {driver, waitForDom} = clientRenderer.render(
+                        <TreeViewDemo />
+                    ).withDriver(TreeViewDemoDriver);
+
+                    const {treeView} = driver;
 
                     const nodeChildren = treeData[0].children;
 
-                    expandItemWithLabel(select, treeData[0].label);
+                    treeView.toggleItem(treeData[0].label);
 
-                    await waitForDom(() => expect(select(getTreeItem(nodeChildren![1].label))).to.be.present());
+                    await waitForDom(() => expect(treeView.getItem(nodeChildren![1].label)).to.be.present());
 
-                    selectItemWithLabel(select, nodeChildren![1].label);
+                    treeView.selectItem(nodeChildren![1].label);
 
                     await waitForDom(() => expect(
-                        isElementFocused(select(getTreeItem(nodeChildren![1].label))!, treeNodeStyle)).to.equal(true));
+                        isElementFocused(treeView.getItem(nodeChildren![1].label)!, treeNodeStyle)).to.equal(true));
```

Comment:
seems like in most cases `isElementFocused` is asked on a tree item. If so `isFocused():boolean` method in `TreeItemDriver` could do the job